src/lib/pipeline/m2/index.js

```diff
@@ -111,7 +120,9 @@ class M2 extends THREE.Group {
 
       geometry.faceVertexUvs = [uvs];
 
-      const mesh = new Submesh(id, geometry, textureUnits);
+      const isBillboard = indexedBones[submesh.rootBone].userData.isBillboard === true;
+
+      const mesh = new Submesh(id, geometry, textureUnits, isBillboard);
```

Comment:
So far, I haven't really needed to use anything directly from submesh.

I think I'd still have to check on the billboard status for the submesh via the M2, since it requires looking up the rootBone.